eng/targets/Helix.Common.props

```diff
@@ -22,7 +22,7 @@
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64'">
     <HelixAvailableTargetQueue Include="Ubuntu.1804.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Windows.10.Amd64.Open" Platform="Windows" />
-    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="OSX" />
+    <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="OSX" />
```

Comment:
In release/6.0 and main, we test on 10.15 in the full matrix (scheduled) runs. Does not seem necessary here. Any objections @dotnet/aspnet-build❔

Reply:
Fine by me